ffmpeg/videoprofile.go

```diff
@@ -36,6 +36,25 @@ const (
 	GOPInvalid = -2
 )
 
+// for sceneclassification
+const (
+	DnnClassify = iota
+	DnnYolo
+	DnnMasking
+)
+
+type DetectorProfile struct {
+	Dnntype    int  //classification or object detect
+	SampleRate uint //execute dnn filter per samplerate
+	ModelPath  string
+	Threshold  float32
+	Input      string
+	Output     string
+	Gpuid      int
```

Comment:
Do we need `Gpuid`? Any calls to the `Transcode()` API should already be associated with a specific GPU so we could just run the DNN filter on the GPU that has already been selected.